File: h2o-docs/src/booklets/v2_2015/source/GLM_Vignette_code_examples/glm_tweedie_example.py
import h2o
from h2o.estimators.glm import H2OGeneralizedLinearEstimator
h2o.init()
h2o_df = h2o.import_file("http://h2o-public-test-data.s3.amazonaws.com/smalldata/glm_test/auto.csv")
tweedie_fit = H2OGeneralizedLinearEstimator(family = "tweedie")
tweedie_fit.train(y = "y", x = h2o_df.col_names[1:], training_frame = h2o_df)

File: h2o-docs/src/booklets/v2_2015/source/GLM_Vignette_code_examples/glm_tweedie_example.R
library(h2o)
h2o.init()
library(HDtweedie)
data(auto) # 2812 policy samples with 56 predictors

dim(auto$x)
hist(auto$y)

# Copy the R data.frame to an H2OFrame using as.h2o()
h2o_df = as.h2o(auto)
vars= paste("x.",colnames(auto$x),sep="")
tweedie.fit = h2o.glm(y = "y", x = vars, training_frame = h2o_df, family = "tweedie")